python/ray/tune/tests/test_trial_runner_2.py

```diff
@@ -133,43 +132,6 @@ def testFailureRecoveryEnabled(self):
         self.assertEqual(len(searchalg.errored_trials), 0)
         self.assertEqual(len(scheduler.errored_trials), 0)
 
-    def testFailureRecoveryNodeRemoval(self):
-        # Node removal simulation only works with resource requests
-        os.environ["TUNE_PLACEMENT_GROUP_AUTO_DISABLED"] = "1"
-
```

Comment:
is there a similar test that we can add?

Reply:
`test_trial_requeue` is the one mimicking node failure.  It's currently failing. Will make sure that one passes before merging this in.